src/generators/static-theme.ts

```diff
@@ -226,7 +226,7 @@ function upperCaseToCamelCase(text: string) {
 export function parseStaticThemes($themes: string) {
     return parseSitesFixesConfig<StaticTheme>($themes, {
         commands: staticThemeCommands,
-        getCommandPropName: upperCaseToCamelCase,
+        getCommandPropName: upperCaseToCamelCase as any,
```

Comment:
I will avoid this `any` cast by restricting the value to `keyof StaticTheme` in a follow-up patch similarly to how it's done elsewhere in the code. I ommitted this change in this patch to keep LoC reasonable.

Reply:
This is done now.

Reply:
Gracious!